packages/unit-jest/README.md

```diff
@@ -284,6 +284,10 @@ This AE ships with project-level type-checking disabled by default since it incr
 
 Project-wide type-checking already takes place during development and into your IDE, so you won't usually need this during tests anyway.
 
+##### SWC support
+
+Experimental support has been added for [swc/jest](https://swc.rs/docs/usage/jest), which is a drop-in Rust replacement for ts-jest and should make your TypeScript Jest tests run faster. To install it, choose the relevant option when prompted on installation, ensuring also to choose typescript.
```

Comment:
Is the SWC option meant to only pop out when you select TS support?
I think we can do so by adding a new question and making the SWC one conditionally bound to the TS one
Is it possible to use SWC for both JS and TS with the same configuration?

Reply:
I wanted it to only popup for typescript but the prompts API doesn't seem to support such an optional question. So currently it does nothing if typescript isn't also selected.

Reply:
IIRC Quasar uses [Inquirer](https://github.com/SBoudrias/Inquirer.js#readme) to process these prompts
You should be able to define a "when" option to control when to show a question

If you aren't able to do so, I'll take a look when you set the PR as ready to review